### DISCUSSION TITLE
Issue : file_get_contents(C:/wamp64/www/folder path): Failed to open stream: No such file or directory

### DISCUSSION THREAD

**Discussion by MOHAMMED-BE** [symfony/symfony#58441] [Q&A]
In Symfony 6.4.*, when I make 'php bin/console make:entity Product' or some other entity, it works, but  when I try it with some other entity, I get the following error : 
php bin/console make:entity Variant
 Your entity already exists! So let's add some new fields!

In FileManager.php line 111:
                                                                                                         
  Warning: file_get_contents(C:/wamp64/www/apiissue/): Failed to open stream: No such file or directory  
                                                                                                         
make:entity [-a|--api-resource] [-b|--broadcast] [--regenerate] [--overwrite] [--with-uuid] [--with-ulid] [--] [<name>]

Note : I have the some project with 6.2, it works correctly.
![Screenshot 2024-10-03 110028](https://github.com/user-attachments/assets/aef0ecd7-4585-4db3-bedb-e9621ecb7a31)

**Comment by programersi-lead** [comment]
I sticking for the topic... the same situation with symfony 6.4.x but only in one project..

**Comment by programersi-lead** [comment]
Ok, i found fix for this ... In my opinion this is a generator bug . 

\vendor\symfony\maker-bundle\src\Generator.php  lines from 155 to 159
are 
``  
       try {
                Validator::classDoesNotExist($className);
                $className = rtrim($fullNamespacePrefix, '\\').'\\'.$className;
            } catch (RuntimeCommandException) {
            }
``
they should be
``
  try {
                $className = rtrim($fullNamespacePrefix, '\\').'\\'.$className;
                Validator::classDoesNotExist($className);
            } catch (RuntimeCommandException) {
            }
``

without this fix  Generator checks class exist in expected sources ... if any other class exist it returns exception and omit $className prefixing. 

in my case, editing the Attribute class will thorow excpetion (because Attribute is exist in global namespace)

**Reply by MOHAMMED-BE** [reply]
I'll try your solution.

My solution:
Downgrade the Symfony Maker Bundle to an older version. 
this works without any issue: composer require symfony/maker-bundle:^1.52

**Reply by MOHAMMED-BE** [reply]
Great.
But the Attribute or any Entity name should works. try var_dump($className) after '$className = rtrim($fullNamespacePrefix...' you'll see that it's return 'string(20) "App\Entity\Attribute"' so that will created as an Entity without throw error of another global namespace.

anyway it's a great idea.

**Reply by programersi-lead** [reply]
In my case before change lines in Generator.php $className return only Attribute without namespace prefix .. so maybe this is really os problem (I work on windows x64 with MAMP)

**Reply by MOHAMMED-BE** [reply]
Yeah, before change the line, it return only Attribute, that why it throw the error of global namespace.

Win64/WAMP.

**Comment by stof** [comment]
I suggest reporting the issue to the MakerBundle, which is maintained in https://github.com/symfony/maker-bundle/

**Reply by MOHAMMED-BE** [reply]
I've already sent a report on this.
https://github.com/symfony/maker-bundle/issues/1609
